arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/halt/SelfdestructSection.java

```diff
@@ -275,6 +274,11 @@ public void resolvePostTransaction(
                   accountBeforeSelfDestruct,
                   selfdestructorAccountAfter.wipe(),
                   DomSubStampsSubFragment.selfdestructDomSubStamps(hub));
+
+      // TODO: ensure this is the right place for this call
+      final DeploymentInfo deploymentInfo = hub.transients().conflation().deploymentInfo();
+      deploymentInfo.updateDeploymentInfoForSelfdestructsAtTransactionEnd(this.recipientAddress);
```

Comment:
The update must happen at transaction end, and this is done in the `resolvePostTransaction` method. So this should be fine. This `updateDeploymentInfoForSelfdestructsAtTransactionEnd` method was renamed in my latest PR.